src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <ProjectGuid>{9C524CA0-92FF-437B-B568-BCE8A794A69A}</ProjectGuid>
     <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">netcoreapp3.0</PackageTargetFramework>
```

Comment:
Let's make this `netcoreapp2.0` and put a comment that says: use this reference assembly only on .NETCoreApp, on desktop and other frameworks use the lib.